jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/Invocable.java

```diff
@@ -29,7 +29,7 @@
  */
 public interface Invocable
 {
-    static ThreadLocal<Boolean> __nonBlocking = new ThreadLocal<>();
+    ThreadLocal<Boolean> __nonBlocking = new ThreadLocal<>();
```

Comment:
Why non-`static`? I don't think it's correct.

Reply:
It's in an interface. It is always static.  Just getting rid of an intellij warning